### PR TITLE
Add contextual links to Security/Backup cards

### LINKED ISSUE #9565: Dashboard: Add contextual links to Security/Backup cards
Currently, if you have a paid Jetpack plan, you'll see cards in your dashboard/AAG that look something like this:

screen shot 2018-05-17 at 9 29 40 am

We should add the contextual link/panels at the bottom (with the arrow, similar to the ones seen for plugins and spam below), and link them with:

**Security**
Text: View security scan details >
Link: `https://dashboard.vaultpress.com/{site}/security/`

**Backups**
Text: View backup history >
Link: `https://dashboard.vaultpress.com/{site}/backups/`

It would look something like:

screen shot 2018-05-17 at 9 34 25 am

### PR DESCRIPTION
**Pull request by Rahmon**
Fixes #9565 

#### Changes proposed in this Pull Request:

* Add contextual links to Security/Backup cards

### Screenshots
Before:
image

After:
screenshot from 2018-05-27 08-34-43

#### Testing instructions:
* Manually on Chrome
* yarn test-client  and yarn test-gui

### CODE REVIEW

**Comment by jeherve** [comment]
Do you think you could add some screenshots about how things should look like, before and after?

Thank you!

**Comment by Rahmon** [comment]
I've a question about the URLs . In the issue, it's suggested for security card https://dashboard.vaultpress.com/{site}/security/ but I don't know exactly what's {site} value should be.

File: _inc/client/at-a-glance/scan.jsx [hunk 1]
```diff
@@ -173,6 +174,17 @@ class DashScan extends Component {
...
 										feature: 'rewind',
 									} )
+
+								}
+								{
+									<Card
+										key="security-scanning"
+										className="jp-dash-item__manage-in-wpcom"
+										compact
+										href={ `https://dashboard.vaultpress.com/${ this.props.siteRawUrl }/security/` }
```
Comment (jeherve) [inline _inc/client/at-a-glance/scan.jsx]: `this.props.siteRawUrl` won't work here. The VaultPress dashboard uses a VaultPress site ID, an   integer that is only used for the VaultPress dashboard.

`site_id` is part of `data`, that is fetched with the `jetpack/v4/module/vaultpress/data` endpoint, so you should be able to grab it from with `vaultPressData`.

Comment (Rahmon) [reply _inc/client/at-a-glance/scan.jsx]: Thank you. I did a new commit.

**Comment by oskosk** [comment]
There's a problem with the last commit. It doesn't address the issue. The proper  name for the site id is `data.site_id`. So

https://github.com/Automattic/jetpack/pull/9646/files#diff-4c277f771dcb6f7dbdd8d55510dd8095R146

and https://github.com/Automattic/jetpack/pull/9646/files#diff-513df2c87e01ded5d25cd014fce87898R163

...both should read `const siteId = data && data.site_id`;

**Comment by Rahmon** [comment]
thanks @oskosk

**Comment by oskosk** [comment]
@Rahmon  If you want to update the PR, then we'll test and include in the next release

**Pull request merged**